cmd/agent.go

```diff
@@ -433,6 +434,7 @@ func (ar *AgentRunner) runInstance() error {
 					Findings:     &res.Findings,
 					Risks:        &res.Risks,
 					Logs:         &res.Logs,
+					TTL:          time.Now().Add(1 * time.Minute).Add(24 * time.Hour), // TODO: hard-coded for demo. Composed of 1 min schedule (plugin-defined) + 24 hrs buffer (agent-defined)
```

Comment:
Should we keep this similar to domain.Observation `Expires` field ? https://github.com/compliance-framework/configuration-service/blob/main/domain/result.go#L226

`TTL` reads "This is how long it is active for".
`Expires` reads "Active until this time"